docs/WifiPerformance.md

```
@@ -59,1362 +59,64 @@ All wireless adapters were tested under consistent conditions - each positioned
 
-## Devices Under Tests
-This section presents the performance test results, including key metrics and technical details from the test execution.  
-**Test Date:** [2025-08-22 22:29 UTC](https://github.com/armbian/armbian.github.io/actions/runs/17166681702)
-### AC
+## WiFi performance
+
+_Measured 2026-07-01 05:19 UTC · [run](https://github.com/armbian/autotests/actions/runs/28494640010)_
+
+**32** wireless link(s) · ✅ **32** pass · 📶 peak ↑ **838** / ↓ **910** Mbps
+
+#### IEEE 802.11ac
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)